sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -383,8 +441,14 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 		log.G(ctrCtx).WithError(err).Error("failed to get container task exit status")
 		return err
 	}
+
 	log.G(ctrCtx).WithField("code", code).Info("container task exited")
 
+  // Return error if container exists with non-zero status
+  if (code != 0) {
+    return errors.New("Container exited with non-zero status")
+  }
```

Comment:
No parentheses are used around simple expressions.  Indentation is also off; should be fixed automatically by `gofmt`.

```suggestion
 	if code != 0 {
 		return errors.New("Container exited with non-zero status")
 	}
```

Reply:
Fixed, `go fmt` fixed that for me :+1: